ui/app/styles/components/replication-dashboard.scss

```diff
@@ -0,0 +1,71 @@
+.replication-dashboard {
+  .has-margin-bottom {
+    padding-bottom: 9px;
+  }
+
+  .has-large-margin-top {
+    margin-top: 40px;
+  }
+
+  .selectable-card {
+    line-height: normal;
+
+    .title-number {
+      font-size: 28px;
```

Comment:
i think the hardcoded values here should be replaced with the variables inside our spacing util here: https://github.com/hashicorp/vault/blob/master/ui/app/styles/utils/_spacing.scss

this way we have consistent spacing throughout the app 😄 

might be worth revisiting the other variables defined in this directory to see if those can be used as well!

Reply:
I was originally trying to use bulma variables, but none of them worked for the designs.  But let me add them to the _spacing doc! I didn't know about that guy.  Thanks!